### DISCUSSION TITLE
MikroORM (MicroORM)

### DISCUSSION THREAD

**Discussion by amzhang** [mikro-orm/mikro-orm#3393] [Ideas]
I know this is somewhat stupid: I pronouced "MikroORM" as "Mee-cro-ORM". Then it dawned on me that it should probably be pronounced as "Micro-ORM".

I think @B4nan should make it clear how to pronounce it in all documentation and blog posts. It's hard to remember the name if it's pronounced "Mee-cro-ORM", and there are plenty of simpletons out there like me :)

Just a thought.

**Comment by B4nan** [comment]
I dont have any preference, mixing this myself as well :] I usually go with the english version if I speak in english.

**Comment by amzhang** [comment]
I only mention it because, as silly as it sounds,having an easy to remember name can improve the popularity of this lib. It helps to occupy more "mind space" if it's east to say/remember.

This is an incredibly well designed library and it should be getting way more attention!
